src/screens/AdjustStock/index.tsx

```diff
@@ -87,21 +87,17 @@ const AdjustStock = () => {
     const navigation = useNavigation();
     const location = useSelector((state: RootState) => state.mainReducer.currentLocation)
     console.log(JSON.stringify(item))
+    const [open, setOpen] = useState(false);
```

Comment:
Why these were moved out from the state below? Additionally, if these needs to be here, should it not have a type?